src/puppet-dom.js

```diff
@@ -50,7 +35,7 @@
       this.element.removeEventListener('puppet-redirect-pushstate', this.historyHandler);
     };
 
-    options.remoteUrl = options.remoteUrl || getRemoteUrlFromCookie() || window.location.href;
+    options.remoteUrl = options.remoteUrl || window.location.href;
```

Comment:
Do we need this line, as `Puppet` constructor (`PuppetNetworkChannel`) , sets it's own default https://github.com/PuppetJs/PuppetJs/blob/e95f936484a9715694ff58f4101daa930a9c83e4/src/puppet.js#L238

If we will remove it, we will get rid of slightly redundant code, but if `remoteUrl` is not provided `puppet.remoteUrl == undefined` and `puppet.network.remoteUrl = new URL(window.location.href)`